cmd/renterd/config.go

```diff
@@ -267,12 +251,6 @@ func parseYamlConfig(cfg *config.Config) {
 }
 
 func parseCLIFlags(cfg *config.Config) {
-	// deprecated - these go first so that they can be overwritten by the non-deprecated flags
-	flag.StringVar(&cfg.Log.Database.Level, "db.logger.logLevel", cfg.Log.Database.Level, "(deprecated) Logger level (overrides with RENTERD_DB_LOGGER_LOG_LEVEL)")
```

Comment:
These deprecated fields are mentioned in the `README.md`. Perhaps we can merge https://github.com/SiaFoundation/renterd/pull/1504 and merge in `dev` or merge that into here.

Reply:
updated it